ClientApp/src/pages/SubmitCombo.jsx

```diff
@@ -275,7 +275,7 @@ export function SubmitCombo() {
             const newMessage = Object.values(apiData.errors).join(' ')
             setErrorMessage(newMessage)
           } else {
-            history.push('/')
+            history.push('/character/' + characterSelected.variableName)
```

Comment:
Please concatenate the string using `/character/${characterSelected.variableName}`